model/mrd_intermediate.yml

```diff
@@ -0,0 +1,88 @@
+NoiseCovariance: !record
+  fields:
+    # Comes from Header.acquisitionSystemInformation.coilLabel
+    coilLabels: CoilLabelType*
+    # Comes from Header.acquisitionSystemInformation.relativeReceiverNoiseBandwidth
+    receiverNoiseBandwidth: float
+    # Comes from Acquisition.sampleTimeUs
+    noiseDwellTimeUs: float
+    # Number of samples used to compute matrix
+    sampleCount: size
+    # Noise covariance matrix with dimensions [coil, coil]
+    matrix: complexfloat[,]
+
+MinMaxStat: !record
+  fields:
+    minimum: uint
+    maximum: uint
+
+AcquisitionBucketStats: !record
+  fields:
+    kspaceEncodeStep1: MinMaxStat
+    kspaceEncodeStep2: MinMaxStat
+    average: MinMaxStat
+    slice: MinMaxStat
+    contrast: MinMaxStat
+    phase: MinMaxStat
+    repetition: MinMaxStat
+    set: MinMaxStat
+    segment: MinMaxStat
+
+AcquisitionBucket: !record
+  fields:
+    data: Acquisition*
+    ref: Acquisition*
+    datastats: AcquisitionBucketStats*
+    refstats: AcquisitionBucketStats*
+    waveforms: WaveformUint32*
+
+# Sampled range along RO, E1, E2 (for asymmetric echo and partial fourier)
+SamplingLimits: !record
+  fields:
+    ro: LimitType
+    e1: LimitType
+    e2: LimitType
+
+SamplingDescription: !record
+  fields:
+    encodedFOV: FieldOfViewMm
+    reconFOV: FieldOfViewMm
+    encodedMatrix: MatrixSizeType
+    reconMatrix: MatrixSizeType
+    samplingLimits: SamplingLimits
+
+BufferedData: !record
+  fields:
+    # Buffered Acquisition data
+    data: complexfloat[loc, s, n, chan, e2, e1, e0]
+    # Buffered Trajectory data
+    trajectory: float[loc, s, n, e2, e1, basis, samples]
+    # Buffered Density weights
+    density: float[loc, s, n, e2, e1, e0]?
+    # Buffered AcquisitionHeaders
+    headers: AcquisitionHeader[loc, s, n, e2, e1]
+    # Sampling details for these Acquisitions
+    sampling: SamplingDescription
+
+ReconBit: !record
```

Comment:
This is the only thing that I don't like......I didn't like it in the Gadgetron either. Some alternatives could be `ReconAssembly` or `ReconPackage` but I am not sure that helps a whole lot.